### DISCUSSION TITLE
[VOTE] Release Apollo version 1.9.0

### DISCUSSION THREAD

**Discussion by lepdou** [apolloconfig/apollo#3899] [General]
Hi All,

This is a call for vote to release Apollo version 1.9.0

Release notes:

 * https://github.com/ctripcorp/apollo/releases/tag/v1.9.0

Release Candidate:

 * https://github.com/ctripcorp/apollo/releases/download/v1.9.0/apollo-configservice-1.9.0-github.zip
 * https://github.com/ctripcorp/apollo/releases/download/v1.9.0/apollo-adminservice-1.9.0-github.zip
 * https://github.com/ctripcorp/apollo/releases/download/v1.9.0/apollo-portal-1.9.0-github.zip
 * sha1 checksums
   - 11145c525c4e83e446da319c8f6560fc9287372c  apollo-configservice-1.9.0-github.zip
   - ed5f3dda5cebe168ac243df2c9ef0ea7ed138171  apollo-adminservice-1.9.0-github.zip
   - 24de71689c9593c1a67b82c46378a64c17e2a06a  apollo-portal-1.9.0-github.zip

Maven 2 staging repository:

- https://oss.sonatype.org/service/local/repositories/comctrip-1021/content/com/ctrip/framework/apollo/apollo-client/1.9.0/apollo-client-1.9.0.pom

Release Tag :

 * https://github.com/ctripcorp/apollo/releases/tag/v1.9.0

Release CommitID :

 * https://github.com/ctripcorp/apollo/commit/772d2fa862b6622bbeb31b50437680887a35e749

Guide to build the release from source :

 * https://www.apolloconfig.com/#/zh/deployment/distributed-deployment-guide?id=_2212-%e9%80%9a%e8%bf%87%e6%ba%90%e7%a0%81%e6%9e%84%e5%bb%ba

Voting will start now (2021/08/16) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by nobodyiam** [comment]
+1 binding to release 1.9.0

I've checked 
- [x] shasum are correct for each zip file
- [x] the apollo-client jars could be fetched from maven repository
- [x] deployed the services and test the main paths without problems
- [x] verified the upgrade process from 1.8.x to 1.9.0

**Comment by klboke** [comment]
+1

**Comment by vdiskg** [comment]
has the `apollo-client-config-data` been published to the maven repository?

**Reply by lepdou** [reply]
Because apollo-client-config-data depends on apollo-client, so publishing apollo-client-config-data should wait until apollo-client has been published to the repo.

**Comment by Anilople** [comment]
+1

**Comment by zouyx** [comment]
+1 binding to release 1.9.0

I've checked 
- [x] All uts passed.
- [x] Download links are fine.
- [x] Shasum are correct.

**Comment by vdiskg** [comment]
+1

**Comment by JaredTan95** [comment]
+1 for me.

- compiled and package from source code.

**Comment by lepdou** [comment]
The vote to release Apollo version 1.9.0 has passed with two +1 binding, 5 +1 and zero -1.  

Thank you~
